assets/js/components/ClustersList.jsx

```diff
@@ -6,7 +6,7 @@ import { addTagToCluster, removeTagFromCluster } from '@state/clusters';
 import ClusterLink from '@components/ClusterLink';
 import { ExecutionIcon } from '@components/ClusterDetails';
 import { ComponentHealthSummary } from '@components/HealthSummary';
-import { axiosPost, axiosDelete } from '@lib/network';
+import { post, del } from '@lib/network';
```

Comment:
nitpick: `del`-`ete` :smile: 
Anyway, as you wish.

Reply:
Actually I was gonna tell you the same :joy: but really, your call

Reply:
It's a reserved word :/

Reply:
aaah damn you're right :frowning:

Reply:
oh noes!
Whatever it works.

Reply:
Leave it as it is, we can refactor it later if we find a way, no worries!